materialize-elasticsearch/type_mapping.go

```diff
@@ -148,13 +147,16 @@ func (constrainter) NewConstraints(p *pf.Projection, deltaUpdates bool) *pm.Resp
 	switch {
 	case p.IsPrimaryKey:
 		constraint.Type = pm.Response_Validated_Constraint_LOCATION_REQUIRED
-		constraint.Reason = "Primary key locations are required"
```

Comment:
This `NewConstraints` function is now exactly the same as the one used in `materialize-sql`: https://github.com/estuary/connectors/blob/7f596b091b0614b01a4bf786b8e43abd2e4ddc69/materialize-sql/type_mapping.go#L414-L453